lib/Web.js

```diff
@@ -126,32 +127,35 @@ function sendNotifications(notifs) {
     notifs.forEach(function (notif) {
         var users = notif.users,
             androidPayload = notif.android,
-            iosPayload = notif.ios,
-            target;
-
-        if (androidPayload && iosPayload) {
-            target = 'all'
-        } else if (iosPayload) {
-            target = 'ios'
-        } else if (androidPayload) {
-            target = 'android';
-        }
+            iosPayload = notif.ios;
+
```

Comment:
all of this should be refactored where we detect the type in the send function and then either delegate to the push `send` function.

and when a different version of this code is in the send we should make sure we are more clear on the decision to try android or try ios.  here it seems like it always tries android via twilio (`redirectToTwilio`) for every request but I am assuming most twilio requests will not result in a push since the twilio service will filter by the type android?

this service should probably look up the device db record first then look at the type and delegate to the local apn method or to the downstream twilio/android(for now) service.  this seems to always try both but relies on only one succeeding. (unless I am mis reading the code)

Reply:
Done